packages/paste-core/components/description-list/src/DescriptionListTerm.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface DescriptionListTermProps extends Omit<React.ComponentPropsWithRef<'div'>, 'children'> {
+  children?: React.ReactNode;
+  element?: BoxProps['element'];
+}
+
+const DescriptionListTerm = React.forwardRef<HTMLElement, DescriptionListTermProps>(
+  ({element = 'DESCRIPTION_LIST_TERM', ...props}, ref) => {
+    return (
+      <Box as="dt" ref={ref} element={element} fontWeight="fontWeightSemibold">
```

Comment:
question: since we're using Box for all of these (dt,dl,dd) without passing any styles, they may render differently on different browsers. Worth trying to see how they look, spacing and font wise, on Safari Chrome FF and Edge if you have it.  

Alternatively we can hardcode our tokens as props here by passing default values.
`<Box as="dt" color="colorText" fontSize="fontSize30" lineHeight="lineHeight30" margin="space0" fontWeight="fontWeightSemibold">`

Reply:
I tested on Safari, Chrome, and FF, and there are no differences I can see, but I'll add the styles just in case the default designs change in the future